app/lib/common/widget/emptyMessagesPlaceholder.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+class EmptyPlaceholder extends StatelessWidget {
+  const EmptyPlaceholder({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      mainAxisAlignment: MainAxisAlignment.start,
+      children: <Widget>[
+        SizedBox(height: MediaQuery.of(context).size.height / 6),
+        SvgPicture.asset('assets/images/emptyPlaceholder.svg'),
+        const SizedBox(height: 10),
+        const Text(
+          'No Messages here yet ...',
```

Comment:
Localisation!?!

Reply:
Yes regarding that we need to reopen ticket for it as some of the content has been changed since it was implemented. We need to rework on it.